web/modules/custom/eulogin/src/Event/Subscriber/JoinupEuLoginCasAccountLinkEventsSubscriber.php

```diff
@@ -53,11 +56,31 @@ public function setEmailCollisionMessage(CasAccountLinkEmailCollisionEvent $even
     ]);
   }
 
+  /**
+   * Sets a random password on the account..
+   *
+   * @param \Drupal\cas_account_link\Event\Events\CasAccountLinkPostLinkEvent $event
+   *   The CAS Account Link post-linking event object.
+   *
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   *   Thrown if the bundle does not exist or was needed but not specified.
+   */
+  public function setFakePassword(CasAccountLinkPostLinkEvent $event): void {
```

Comment:
Just a small suggestion, but wouldn't it be better to call it `setRandomPassword()`? That would better explain what this method does. Also the password is not really fake, it is still a real password, and even a very good one with 30 characters ;)